es-app/src/main.cpp

```diff
@@ -65,6 +65,9 @@ bool parseArgs(int argc, char* argv[], unsigned int* width, unsigned int* height
 		}else if(strcmp(argv[i], "--no-splash") == 0)
 		{
 			Settings::getInstance()->setBool("SplashScreen", false);
+		}else if(strcmp(argv[i], "--system") == 0)
```

Comment:
Do we want to retain the command line option on top of the UI-based setting?
It does not hurt of course...